README.md

```diff
@@ -27,9 +27,11 @@ Refer to here: https://www.notion.so/opengov/Netlify-to-Amplify-Migration-01b9ba
 
```

Comment:
Who uses this script and when? Can we add to runbook?

Reply:
https://www.notion.so/opengov/On-Call-Runbook-ab4bffa2472d47ef9b657242632da7f9?pvs=4#908fe1bb0e494ebe9493d0c4ab6647ef
is this sufficient?

Reply:
looks good